cumulusci/core/keychain/encrypted_file_project_keychain.py

```diff
@@ -107,3 +114,24 @@ def _raise_service_not_configured(self, name):
             f"'{name}' service configuration could not be found. "
             f"Maybe you need to run: cci service connect {name}"
         )
+
+    def _get_org(self, name):
+        org = self._decrypt_config(
+            OrgConfig,
+            self.orgs[name].encrypted_data,
+            extra=[name],
+            context=f"org config ({name})",
+        )
```

Comment:
It doesn't look like _decrypt_config attaches the keychain to the org config that it constructs.

Reply:
Okay, I think you're right -- but wouldn't it be cleaner to attach the keychain right when the org config is constructed? That way if org config constructors ever need to do something with the keychain, they can.

Reply:
Okay, take a look at: c074c621

I left code in BaseProjectKeychain because not every subclass may implement _get_org in a way that sets the keychain. To make the keychain 100% reliably available for org constructors, we'd need to make it mandatory in the initializer which would be a breaking change.